src/components/HeaderGradeDisplay.tsx

```diff
@@ -18,7 +18,7 @@ const Label = styled.p`
 `;
 const HeaderGradeDisplayContainer = styled.div`
 	background-color: ${({ theme }) => theme.colors.main[0]};
-	width: 15em;
+	width: auto;
```

Comment:
Just curious why this was changed, and if we are in need of a min-width?

Reply:
i set the width in the container the grade display is in, so that the spacing is correct when you resize the window